docs/glossary.md

```diff
@@ -194,6 +194,14 @@ overly prescriptive about how it should be used. Userland enables users to
 create and share tools that provide additional functionality on top of what is
 available in "core".
 
+### utility process
+
+The utility process is a child of the main process that allows running any
+untrusted services that cannot be run in the main process. Chromium uses this
+process to perform network I/O, audio/video processing, device inputs etc.
```

Comment:
Maybe add a link to the UtilityProcess docs here?

Reply:
https://github.com/electron/electron/pull/34980/commits/1b3ac3f62612fa286f265e8ecc191149c9c65ce7